Typo in source code?

Hi,

I'm building with -DWITH_MEDIA_FOUNDATION=ON (MSVC, master / v3.30.0), and it fails to compile libfreerdp/codec/h264_mf.c:
```
h264_mf.c: error C2039: 'YUVHeigth' is not a member of 'S_H264_CONTEXT'
h264_mf.c: error C2039: 'frameHeigth' is not a member of 'H264_CONTEXT_MF'
```

In mf_decompress, right after outputBuffer->lpVtbl->Release(...):
```
h264->YUVHeigth = sys->frameHeigth;   // typo: should be YUVHeight / frameHeight
```
Same misspelling in Android h264_mediacodec.c.
```
Fix: h264->YUVHeight = sys->frameHeight;
```

--- a/libfreerdp/codec/h264_mediacodec.c
+++ b/libfreerdp/codec/h264_mediacodec.c
@@ -357,7 +357,7 @@ static int mediacodec_decompress(H264_CONTEXT* h264, const BYTE* pSrcData, UINT3
 				              iStride[1] * ((sys->outputHeight + 1) / 2);
 
 				h264->YUVWidth = sys->outputWidth;
-				h264->YUVHeigth = sys->outputHeigth;
+				h264->YUVHeight = sys->outputHeight;
 				break;
 			}
 			else if (outputBufferId == AMEDIACODEC_INFO_OUTPUT_FORMAT_CHANGED)

--- a/libfreerdp/codec/h264_mf.c
+++ b/libfreerdp/codec/h264_mf.c
@@ -376,7 +376,7 @@ static int mf_decompress(H264_CONTEXT* WINPR_RESTRICT h264, const BYTE* WINPR_RE
 
 		outputBuffer->lpVtbl->Release(outputBuffer);
 		h264->YUVWidth = sys->frameWidth;
-		h264->YUVHeigth = sys->frameHeigth;
+		h264->YUVHeight = sys->frameHeight;
 	}
 
 	inputSample->lpVtbl->Release(inputSample);

